packages/page-democracy/src/Overview/Referendum.tsx

```diff
@@ -95,7 +96,10 @@ function Referendum ({ className = '', value: { allAye, allNay, image, imageHash
         proposal={image?.proposal}
       />
       <td className='number together media--1200'>
-        <BlockToTime value={remainBlock} />
+        <BlockToTime
+          value={remainBlock}
+
+        />
```

Comment:
```suggestion
        <BlockToTime value={remainBlock} />
```

Actutally weird, I was convinced the JSX linting rule should pick up both -

- extra blank lines before the closing tag
- not having a single attribute on a single line (the reverse is certainly true)